ts/app.ts

```diff
@@ -22,24 +24,35 @@ if (data) {
   id = 0;
 }
 
-function loadList(array) {
-  array.forEach(function (item) {
+function loadList(
+  array: { name: string; id: number; done: boolean; trash: boolean }[]
+) {
+  array.forEach(function (item: {
```

Comment:
same here, export interface... check if you can reuse the upper one

Reply:
done